repositories.yaml

```diff
@@ -12,6 +25,18 @@ glean:
     - 'glean-core/pings.yaml'
   library_names:
     - org.mozilla.components:service-glean
+glean-deprecated:
```

Comment:
Why are we leaving this around?

Reply:
There's some description in this commit: https://github.com/mozilla/probe-scraper/pull/217/commits/1db46bea9dca73a55c384b10b25536cb43203af6

Reply:
Oh, shame on me for not reading the commit message. Sorry!